cmd/govim/watcher.go

```diff
@@ -80,19 +98,19 @@ func (m *modWatcher) watch() {
 	}
 	for {
 		select {
-		case event, ok := <-m.watcher.Events:
+		case event, ok := <-m.watcher.events:
 			if !ok {
 				// watcher has been stopped?
 				return
 			}
-			switch event.Op {
-			case fsnotify.Remove, fsnotify.Rename:
-				path := event.Name
+			switch event.op {
+			case fsOpRemoved:
```

Comment:
Just so you're aware: https://github.com/myitcv/govim/issues/465

Reply:
Ah! I could add a few tests as well. Both for deleting and tests that proves that adding/removing sub-directories with files (since the different implementations handle sub dirs differently).

Reply:
Deleting files currently has no effect in the version of `gopls` we're using, so let's leave those until later.